packages/drivers/routerlicious-driver/src/documentServiceFactory.ts

```diff
@@ -22,15 +22,34 @@ import { DocumentService } from "./documentService";
 import { ITokenProvider } from "./tokens";
 import { RouterliciousOrdererRestWrapper } from "./restWrapper";
 
+export interface IRouterliciousDriverPolicies {
```

Comment:
I think we should add it in a separate file, maybe like policies.ts or utils maybe.